features/validate_image.feature

```diff
@@ -411,6 +411,39 @@ Feature: evaluate enterprise contract
     When ec command is run with "validate image --image ${REGISTRY}/acceptance/ec-happy-day --policy acceptance/ec-policy --public-key ${known_PUBLIC_KEY} --rekor-url ${REKOR}  --show-successes"
     Then the exit status should be 0
     Then the output should match the snapshot
+  
+  Scenario: policy rule filtering on imageRef
+    Given a key pair named "known"
+    Given an image named "acceptance/ec-happy-day"
+    Given a valid image signature of "acceptance/ec-happy-day" image signed by the "known" key
+    Given a valid Rekor entry for image signature of "acceptance/ec-happy-day"
+    Given a valid attestation of "acceptance/ec-happy-day" signed by the "known" key
+    Given a valid Rekor entry for attestation of "acceptance/ec-happy-day"
+    Given a git repository named "happy-day-policy" with
+      | filtering.rego | examples/filtering.rego |
+    Given policy configuration named "ec-policy" with specification
+    """
+    {
+      "sources": [
+        {
+          "volatileConfig": {
+            "include": [
+              {
+                "value": "filtering.always_pass",
+                "imageRef": "sha256:${REGISTRY_acceptance/ec-happy-day:latest_DIGEST}"
+              }
+            ]
+          },
```

Comment:
Perhaps we could also test the exclude case here (it might be more common)

```suggestion
      "sources": [
        {
          "configuration": {
          	"include": [ "filtering.always_fail" ]
          }
          "volatileConfig": {
            "include": [
              {
                "value": "filtering.always_pass",
                "imageRef": "sha256:${REGISTRY_acceptance/ec-happy-day:latest_DIGEST}"
              }
            ],
            "exclude": [
              {
                "value": "filtering.always_fail",
                "imageRef": "sha256:${REGISTRY_acceptance/ec-happy-day:latest_DIGEST}"
              }
            ]
          },
```